frontend/src/features/properties/map/research/common/ResearchMenu.tsx

```diff
@@ -0,0 +1,76 @@
+import { Col, Row } from 'react-bootstrap';
+import { FaCaretRight } from 'react-icons/fa';
+import styled from 'styled-components';
+
+export interface IResearchMenuProps {
+  items: string[];
+  selectedIndex: number;
+  onChange: (index: number) => void;
+}
+
+const ResearchMenu: React.FunctionComponent<IResearchMenuProps> = props => {
+  const handleClick = (index: number) => {
+    props.onChange(index);
+  };
+  return (
+    <StyledMenuWrapper>
+      {props.items.map((label: string, index: number) => (
+        <StyledRow
```

Comment:
I don't see a key here so you may be getting a console warning from react about not having a unique key per mapped row.